src/runtime/test/manifest-parser-test.ts

```diff
@@ -298,4 +298,55 @@ describe('manifest parser', () => {
         in Sloturnicus s
     `);
   });
+  it('does not parse comment at start of manifest resource', () => {
+    let data;
+    assert.throws(() => {
+      const manifestAst = parse(`
+        import '../Pipes/PipeEntity.schema'
+
+        resource PipeEntityResource
+          start
+          //[{"type": "tv_show", "name": "star trek"}]
+          [{"type": "artist", "name": "in this moment"}]
+
+        store ExamplePipeEntity of PipeEntity 'ExamplePipeEntity' @0 in PipeEntityResource
+      `);
+      data = JSON.parse(manifestAst[1].data);
+    }, `Unexpected token / in JSON at position 0`);
+    assert.equal(data, undefined);
+  });
+  it('does not parse comment inside manifest resource', () => {
+    let data;
+    assert.throws(() => {
+      const manifestAst = parse(`
+        import '../Pipes/PipeEntity.schema'
+
+        resource PipeEntityResource
+          start
+          [{"type": "artist", "name": "in this moment"}]
+          //[{"type": "tv_show", "name": "star trek"}]
+
+        store ExamplePipeEntity of PipeEntity 'ExamplePipeEntity' @0 in PipeEntityResource
+      `);
+      data = JSON.parse(manifestAst[1].data);
+    }, `Unexpected token / in JSON at position 47`);
+    assert.equal(data, undefined);
+  });
+  it('ignores comments inside manifest resource', () => {
```

Comment:
Presumably this is actually 'breaks out of parsing resource when encountering comment at different indent level'? The comment isn't ignored in that parsing it ends resource extraction.